app/src/main/java/com/codesoom/assignment/utils/JwtEncoder.java

```diff
@@ -0,0 +1,40 @@
+package com.codesoom.assignment.utils;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.security.Keys;
+import org.springframework.beans.factory.annotation.Value;
+
+import java.security.Key;
+
+/**
+ * JWT 토큰의 생성과, 디코딩을 담당합니다.
+ */
+public class JwtEncoder {
+
+    private final Key key;
+
+    public JwtEncoder(@Value("${jwt.secret}") String secret) {
+        key = Keys.hmacShaKeyFor(secret.getBytes());
+    }
+
+    /**
+     * 유저 id를 입력받아 JWT 토큰을 생성하고 리턴합니다.
+     *
+     * @param id 유저 id
+     * @return JWT Token
+     */
+    public String encode(Long id) {
+        return Jwts.builder().claim("userId", 1L).signWith(key).compact();
```

Comment:
이런 경우는 여러 가지 작업이 한 줄에 전부 들어있어 한 눈에 읽기 어렵습니다. 가장 쉬운 방법은 한 줄에 `.`이 한 개씩 오도록 개행해주는 거에요.

```suggestion
        return Jwts.builder()
            .claim("userId", 1L)
            .signWith(key)
            .compact();
```